src/selection.ts

```diff
@@ -299,6 +295,6 @@ export function isLegendStreamBinding(bind: any): bind is LegendStreamBinding {
   return isLegendBinding(bind) && isObject(bind);
 }
 
-export function isParameterSelection(param: any): param is SelectionDef {
+export function isInteractiveParameter(param: any): param is SelectionParameter {
```

Comment:
I'm not sure why this was changed? Variable parameters can also be interactive? Either the previous name or `isSelectionParameter` makes sense to me.

Reply:
I prefer `isSelectionParameter`.